src/libs/actions/Report.ts

```diff
@@ -2507,46 +2508,20 @@ function getCurrentUserAccountID(): number {
 }
 
 function navigateToMostRecentReport(currentReport: OnyxEntry<Report>) {
-    const reportID = currentReport?.reportID;
-    const sortedReportsByLastRead = ReportUtils.sortReportsByLastRead(Object.values(allReports ?? {}) as Report[], reportMetadata);
-
-    // We want to filter out the current report, hidden reports and empty chats
```

Comment:
**Filtering out the current report:**
This is now part of `findLastAccessedReport`.

**Filtering out hidden reports:**
This is not needed. We should navigate to the recent report no matter its notification preference. I believe this was done initially as a workaround because after you leave a workspace room we just change its notification status and we wanted to avoid landing on the same report. This problem is solved by excluding the current report.

**Filtering out empty reports:**
This is also not needed. I believe this was done as workaround because it was possible that you land on a chat that you never visited before. This is no longer a problem since `sortReportsByLastRead` will filter such reports (reports that do not have `lastReadTime` field).